templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -202,6 +202,17 @@ After which you should see something similar to:
 2016-07-07 15:06:59,187 INFO     127.0.0.1       POST    200 143ms (mem: 6ms/13) (md: 43ms/1) (db: 20ms/9q) (+start: 147ms) /api/v1/external/helloworld (helloworld-bot@zulip.com via ZulipHelloWorldWebhook)
 ```
 
+Using the Integrations Developer Panel (GUI):
+After doing `./tools/run-dev.py` go to http://localhost:9991/devtools/integrations/.
+
+Select an *incoming webhook* bot, an integration and optionally a fixture which you can then further
+optionally tweak before sending. You may also optionally specify a stream and a topic.
+
+`static/images/integrations/integrations_dev_panel.png`
+
+By opening the IDP in one tab and zulipchat in another, you can quickly tweak, send and view many
+different test fixtures.
+
 Using curl:
 
 ```
```

Comment:
I would recommend perhaps splitting this up into two sections, one for curl, and the other for the IDP GUI. :)

Reply:
This is pretty good idea. In fact, it might be good to split up this area up into 3 sections, one for each of the tools. :slightly_smiling_face: